components/ContentContainer/contentContainer.module.scss

```diff
@@ -8,6 +8,7 @@
   border-radius: 5px;
   background-color: white;
   box-shadow: 1px 2px 5px rgb(174, 168, 168);
+  max-width: 750px;
```

Comment:
I set a max-width, for the time-being, for my ContentContainer, because it looked like, from the wireframes, that the page never extends fully across the full width of the view.